### PR TITLE
Fix #4031 - Move source and outline tabs to the top of the left pane

### LINKED ISSUE #4031: [Outline] move the tabs to the top
The new photon designs call for moving the sources/outline tabs to the top to be consistent.

We hopefully can borrow the html/css from the editor tabs, while re-using the JS...

### PR DESCRIPTION
**Pull request by darkwing**
Putting this in quick to see how people like it.

after

### CODE REVIEW

**Comment by jasonLaster** [comment]
:thumbsup:

**Comment by darkwing** [comment]
OK, this is ready for a look.  Down the road we may want to refactor the CSS here because `class="source-footer"` is used twice: once for the editor footer and also for this sources/outline container.  That seems like it could cause trouble down the road...maybe.

File: src/components/PrimaryPanes/index.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ class PrimaryPanes extends Component {
     ];
   }
 
-  renderFooter() {
+  renderTabs() {
     return <div className="source-footer">{this.renderOutlineTabs()}</div>;
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/index.js]: perhaps we want to duplicate the CSS now and move it over to PrimaryPane.css

File: src/components/PrimaryPanes/Sources.css [hunk 2]
```diff
@@ -114,17 +115,26 @@
   cursor: default;
 }
 
+.source-footer .tab:first-child {
+  border-left: 0;
+}
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/Sources.css]: does this get us the border radii he Editor tabs?

**Comment by jasonLaster** [comment]
hmm, i suppose this is a step in the right direction.

**Comment by darkwing** [comment]
Per feedback:

-  There's a screenshot in the PR at the top :D
-  I'll move over the CSS
-  I didn't duplicate tab radius because but I'll give it a shot.  Part of me thinks it will be sort of confusing because they aren't closable and they're persistently there but we can see when it's done.

**Comment by darkwing** [comment]
OK, updated.

leftsourceoutlinelight

leftsourceoutline

**Comment by jasonLaster** [comment]
Thanks @darkwing

**Pull request merged**